pages/deposits_and_withdrawals/cli_commands.md

```diff
@@ -0,0 +1,27 @@
+## CLI commands
+
+### Deposit to Subaccount
+
+```bash
+dydxprotocold tx sending deposit-to-subaccount <sender_key_or_address> <recipient_address> <recipient_subaccount_number> <quantums> [flags]
+```
+
+Example:
+```bash
+NATIVE_TOKEN_DENOM=adv4tnt
```

Comment:
We should avoid spelling out the native token denom explicitly here but rather keep all the info in https://docs.dydx.exchange/networks/network1/network_constants#native-token-denom and then only point to that file.

So here, I would suggest removing this line and just have a line saying something like "see this page for the native token denom".

Reply:
The NATIVE_TOKEN_DENOM var is just for ease of use on the cmd here. I already have the note pointing to the url